client/gutenberg/extensions/map/editor.js

```diff
@@ -0,0 +1,33 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import { registerBlockType } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+
+import { settings } from './settings.js';
+import edit from './edit';
+import save from './save';
+import './style.scss';
+import './editor.scss';
+
+registerBlockType( settings.name, {
+	title: settings.title,
+	icon: settings.icon,
+	category: settings.category,
+	keywords: settings.keywords,
+	attributes: settings.attributes,
```

Comment:
Would [spread](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax) work here?
```
{
  ...settings,
}
```

Although it's not very obvious what would get set so for transparency current way might be better.

Reply:
Since this works, I'm going to mark it as Resolved for now - we can revisit later.